index.d.ts

```diff
@@ -1,30 +1,36 @@
 import * as React from 'react';
-import {default as tippyCore, Instance, Props, Plugin} from 'tippy.js';
+import {
+  default as tippyCore,
+  Instance,
+  Props,
+  Plugin,
+  Placement,
+} from 'tippy.js';
 
-export interface TippyProps extends Omit<Partial<Props>, 'content'> {
+export interface TippyProps extends Omit<Partial<Props>, 'content' | 'render'> {
   content: React.ReactChild | React.ReactChild[];
   children: React.ReactElement<any>;
   visible?: boolean;
-  enabled?: boolean;
+  disabled?: boolean;
   className?: string;
   singleton?: (instance: Instance) => void;
+  render?: (attrs: {
+    'data-placement': Placement;
+    'data-reference-hidden'?: string;
+    'data-escaped'?: string;
+  }) => React.ReactNode;
```

Comment:
This allows for very easy spreading: `render={attrs => <Box {...attrs} />}`. If they want to rename the attributes to something else, that's obviously also still possible, but a little bit unclean with the way it's accessed, e.g. 

```jsx
<Tippy
  render={attrs => (
    <Box
      data-someprefix-placement={attrs['data-placement']}
      data-someprefix-reference-hidden={attrs['data-reference-hidden']}
    />
  )}
/>
```